net-libs/libaccounts_glib/libaccounts_glib-1.27.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="Accounts management library for GLib applications"
+DESCRIPTION="This project is a library for managing accounts which can be used from GLib \
+applications. It is part of the [accounts-sso project][accounts-sso]."
+HOMEPAGE="https://gitlab.com/accounts-sso/libaccounts-glib"
+COPYRIGHT="2024 Accounts SSO"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="https://gitlab.com/accounts-sso/libaccounts-glib/-/archive/VERSION_1.27/libaccounts-glib-VERSION_1.27.tar.bz2"
+CHECKSUM_SHA256="e178c103e60ca34777afba94019a1c4571aedf9e54291b0faca71e5cad0628af"
+SOURCE_DIR="libaccounts-glib-VERSION_1.27"
```

Comment:
maybe use $portVersion here?

Reply:
It should :)

Reply:
Resolved in: https://github.com/haikuports/haikuports/pull/11108